frontend/views/containers/proposals/AddMembers.vue

```diff
@@ -111,67 +99,51 @@ export default {
-          class: 'has-text-success'
-        })
+    inviteeUpdate (e, index) {
+      if (e.target.value.length > 0 && !this.ephemeral.isValid) {
         this.ephemeral.isValid = true
-      } catch (err) {
-        console.log(err)
-        this.setInviteError(index, this.L('Non existing user. TODO support general invites'))
       }
-    },
-    setInviteError (index, text) {
-      this.ephemeral.invitesToSend[index] = null
-      Vue.set(this.form.eachFeedbackMsg, index, {
-        text,
-        class: 'error'
-      })
+      if (e.target.value.length === 0 && this.ephemeral.invitesCount === 1 && this.ephemeral.isValid) {
+        this.ephemeral.isValid = false
+      }
     },
     removeInvitee (index) {
       this.ephemeral.invitesCount -= 1
       this.form.invitees.splice(index, 1)
-      this.ephemeral.invitesToSend[index] = null
     },
-    addInviteeSlot () {
+    addInviteeSlot (e) {
       this.ephemeral.invitesCount += 1
+      Vue.nextTick(() => {
+        const inviteeSlots = this.$refs.fieldset.getElementsByTagName('label')
+        const newInviteeSlot = inviteeSlots[inviteeSlots.length - 1]
+        newInviteeSlot && newInviteeSlot.focus()
+      })
     },
     async submit (form) {
-      try {
+      const invitationsSent = []
+      let hasFailed = false
+      // NOTE: All invitees proposals will expire at the exact same time.
+      // That plus the proposal creator is what we'll use to know
+      // which proposals should be displayed visually together.
+      const expiresDateMs = Date.now() + this.groupSettings.proposals[PROPOSAL_INVITE_MEMBER].expires_ms
+
+      for (const inviteeName of this.form.invitees) {
         const groupId = this.currentGroupId
-        const groupName = this.groupSettings.groupName
-        // NOTE: All invitees proposals will expire at the exact same time.
-        // That plus the proposal creator is what we'll use to know
-        // which proposals should be displayed visually together.
-        const expiresDateMs = Date.now() + this.groupSettings.proposals[PROPOSAL_INVITE_MEMBER].expires_ms
-        for (const index in this.ephemeral.invitesToSend) {
-          const member = this.ephemeral.invitesToSend[index]
-          const mailbox = member.attributes.mailbox
-          const memberName = member.attributes.name
+        let identityContract
+        try {
+          const contractID = await sbp('namespace/lookup', inviteeName)
+          identityContract = await sbp('state/latestContractState', contractID)
+        } catch (e) {
+          // The invitee isn't registered... but it's okay at #614
+        }
 
-          if (this.groupShouldPropose) {
+        if (this.groupShouldPropose) {
+          try {
             const proposal = await sbp('gi.contracts/group/proposal/create',
               {
                 proposalType: PROPOSAL_INVITE_MEMBER,
                 proposalData: {
-                  member: memberName,
+                  member: inviteeName,
```

Comment:
(Putting this comment here because not sure where else to put it).

When the member who was invited through a proposal visits the link to join, there are errors and warnings in the console that need to be fixed:

```
[sbp] state/vuex/state 
Array []
main.js:44:14
latestContractState: ignoring mutation 21XWnNSWL9Y142KLMCsAJbPoTjo1wBrQXRN7Mi5dUoB1xs1QC6|gi.contracts/group/proposal/process because of TypeError state.js:69:18
[sbp] gi.contracts/group/proposalVote/process 
Array [ {…}, {…} ]
main.js:44:14
proposalVote: no proposal for 21XWnNSWL9Y142KLMCsAJbPoTjo1wBrQXRN7Mi5dUoB1xs1QC6! 
Object { data: {…}, meta: {…}, hash: "21XWnNLyrqJoU8viyDrdzMRgmnZiJm1V37So5EsHooLCmoBGYQ" }
group.js:183:18
latestContractState: ignoring mutation 21XWnNLyrqJoU8viyDrdzMRgmnZiJm1V37So5EsHooLCmoBGYQ|gi.contracts/group/proposalVote/process because of GIErrorIgnoreAndBanIfGroup state.js:69:18
[sbp] gi.contracts/group/proposalVote/process 
Array [ {…}, {…} ]
main.js:44:14
proposalVote: no proposal for 21XWnNSWL9Y142KLMCsAJbPoTjo1wBrQXRN7Mi5dUoB1xs1QC6! 
Object { data: {…}, meta: {…}, hash: "21XWnNLHmwSWijBacCSGy1Tff5KXRDP2AKyn54i1ESzsnYXwgh" }
group.js:183:18
latestContractState: ignoring mutation 21XWnNLHmwSWijBacCSGy1Tff5KXRDP2AKyn54i1ESzsnYXwgh|gi.contracts/group/proposalVote/process because of GIErrorIgnoreAndBanIfGroup state.js:69:18
```

This happens on the "You've been invited to join a group!" screen